webkitpy/benchmark_runner/browser_driver/linux_cog_driver.py

```diff
@@ -36,3 +36,11 @@ def launch_url(self, url, options, browser_build_path):
 
     def launch_driver(self, url, options, browser_build_path):
         raise ValueError("Browser {browser} is not available with webdriver".format(browser=self.browser_name))
+
+
+class CogFdoBrowserDriver(CogBrowserDriver):
+    browser_name = 'cog-fdo'
+
+    def launch_url(self, url, options, browser_build_path):
+        self._browser_arguments = ['-P fdo', url]
```

Comment:
The `-P fdo` command line arguments need to be *two* separate items. This needs to be:

```python
self._browser_arguments = ['-P', 'fdo', url]
```

or, event better, as it is more descriptive:

```python
self._browser_arguments = ['--platform=fdo', url]
```